### ISSUE TITLE
ska3-template is broken on osx due to sed POSIX thingy

### ISSUE THREAD

**Issue by jeanconn** [sot/skare3#26]
Basically, I think I need to replace 

```
sed -i "searchandreplacethingy" filename
```
with
```
sed -i "" "searchandreplacethingy" filename
```

I think the explicit empty string is needed for the -i option to the sed command. From the help it is really "sed [-i extension]" and it looks like on linux sed must be smart enough to figure out that the "searchandreplace" command is a command and not a file extension if you neglect to put something for the inplace file extension.

I think this won't need to build on osx if we can noarch the template, but all the recipes should really work.

Without the empty string for -i, on the mac I get something like
```
sed -i "s|%{PREFIX}%|${PREFIX}|" ska_conda/pkg_defs/ska3-template/bin/flt_envs
sed: 1: "ska_conda/pkg_defs/ska3 ...": bad flag in substitute command: 'a'
```

**Comment by jeanconn** [comment]
Might also be able to just put the "-i" after the command.

**Comment by taldcroft** [comment]
I just looked and on linux the `-i` needs to be followed by the replacement character(s) with no space, i.e. `-i.bak`.

**Comment by jeanconn** [comment]
Gotcha, which explains why it was working on linux (not magically understanding that my command string wasn't a file extension, just no characters after the -i).

**Comment by jeanconn** [comment]
And which probably also means I didn't test #28 correctly on linux.  I'll make sure I have a works-on-both solution in the PR before I'm done.

**Comment by taldcroft** [comment]
Closed by #28 (which eliminates sed entirely)
